src/lwc/geFormField/geFormField.js

```diff
@@ -135,7 +135,9 @@ export default class GeFormField extends LightningElement {
         let fieldIsValid = this.checkFieldValidity();
 
         if(this.element !== null && this.element.required) {
-            return isNotEmpty(this.value) && fieldIsValid;
+            return ( isNotEmpty(this.value) &&
```

Comment:
Do we need the parentheses around the first two conditions, since they are all && ?

Reply:
fixed!